lib/src/channel_manager.dart

```diff
@@ -19,13 +19,11 @@ class ChannelManager {
     await _channel.invokeMethod('animate', animate);
   }
 
-  static Future<bool> expand() async {
-    return await _channel.invokeMethod('expand');
-  }
+  static Future<bool> expand() =>
+      _channel.invokeMethod<bool>('expand').then((x) => x ?? false);
 
-  static Future<bool> fling(double velocity) async {
-    return await _channel.invokeMethod('fling', velocity);
-  }
+  static Future<bool> fling(double velocity) =>
+      _channel.invokeMethod<bool>('fling', velocity).then((x) => x ?? false);
```

Comment:
i'm a little out-of-date on dart language updates. Like `Observables` used in JS, I guess `Future`s are now returned by synchronous methods?

makes sense to me.

Reply:
This has always been possible in Dart and it's slightly more concise for one-liners like this. 
`_channel.invokeMethod<bool>` returns a `Future<bool?>` and the `.then((x) ...` maps the output of `invokeMethod` to a non-nullable bool to match the return type of `fling`